events.css

```diff
@@ -0,0 +1,29 @@
+article section: first-child {
```

Comment:
Be sure to delete the white space after the ":", as this was causing the styling to not show up on the first child of Section.